src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemFilerTests.cs

```diff
@@ -45,12 +193,24 @@ public void WorkItemFiler_ValidatesSarifLogFileLocationArgument()
             action.Should().Throw<ArgumentNullException>();
         }
 
-        private static SarifWorkItemFiler CreateWorkItemFiler()
-            => CreateMockSarifWorkItemFiler();
+        private static SarifWorkItemFiler CreateWorkItemFiler(SarifWorkItemContext context = null)
+            => CreateMockSarifWorkItemFiler(out _, context).Object;
```

Comment:
>out _ [](start = 44, length = 5)

out parameters last.

Reply:
But the mock filer and the mock client are independent. Why not a separate method CreateMockFilingClient? 

---
In reply to: [389390477](https://github.com/microsoft/sarif-sdk/pull/1800#discussion_r389390477) [](ancestors = 389390477)